include/dlaf_c/grid.h

```diff
@@ -0,0 +1,24 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) 2018-2023, ETH Zurich
+// All rights reserved.
+//
+// Please, refer to the LICENSE file in the root directory.
+// SPDX-License-Identifier: BSD-3-Clause
+//
+
+#pragma once
+
+#include <mpi.h>
+
+#include "dlaf_c/utils.h"
+
+DLAF_EXTERN_C MPI_Comm Cblacs2sys_handle(int ictxt);
+DLAF_EXTERN_C void Cblacs_get(int ictxt, int what, int* val);
+DLAF_EXTERN_C void Cblacs_gridinfo(int ictxt, int* np, int* mp, int* px, int* py);
```

Comment:
These can maybe be put in a separate header included only for the internal implementations? We wouldn't want users to rely on us declaring these for them.

Reply:
Good suggestion. I'm still debating if we should include `mkl_blacs.h` for MKL and define them only for other implementations. Or just do what COSMA seems to do and define them regardless.

Reply:
> These can maybe be put in a separate header included only for the internal implementations?

Done. But I'll leave this unresolved until I have checked if for MKL we cam/should just use `mkl_blacs.h`.